lib/src/components/NxFilterInput/NxFilterInput.tsx

```diff
@@ -23,16 +26,40 @@ const NxFilterInput = forwardRef<HTMLDivElement, Props>(
           className = classnames('nx-filter-input', classNameProp, {
             'nx-filter-input--empty': isEmpty
           }),
-
+          btnClassName = classnames('nx-btn--clear', { 'hidden': isEmpty }),
+          fieldRef = useRef<HTMLDivElement>(null),
+          mergedRef = useMergedRef(fieldRef, ref),
           // just in case these props get passed in, avoid passing them to NxTextInput as they would cause
           // malfunction
           cleanedProps = omit(['validatable', 'validationErrors', 'type'], otherProps),
           filterIcon = searchIcon ? faSearch : faFilter,
-          prefixContent = <NxFontAwesomeIcon icon={filterIcon} className="nx-icon--filter-icons" />;
+          prefixContent = <NxFontAwesomeIcon icon={filterIcon} className="nx-icon--filter-icons" />,
+          suffixContent =
+            <NxButton className={btnClassName}
+                      variant="icon-only"
+                      title="clear filter"
```

Comment:
Should the title be "clear search" when `searchIcon` is true?

Reply:
It makes sense to me but will confirm with design.

Reply:
Confirmed with designer @JasonDSwearingen that it should read "Clear search" when `searchIcon` is true. 
Updated mockups https://www.sketch.com/s/2393264e-1233-4909-99c8-a7dda02588cd/a/g0GK9mj.
Updated title 7037e5e100f7b9e7829500a1b141c7b9a7ded913
Updated unit tests 6db707ba13ca7b6bbaba1769b0d74d0551a74628
Updated text with sentence case to match final mockups 506f08770df82ddf592cfc324fa6aa904516f5d0